README.md

```diff
@@ -10,22 +10,105 @@
 [![Chat](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ManageIQ/manageiq-providers-redfish?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Translate](https://img.shields.io/badge/translate-zanata-blue.svg)](https://translate.zanata.org/zanata/project/view/manageiq-providers-redfish)
 
-ManageIQ plugin for the Redfish provider.
 
-## Development
+## Quickstart
 
-See the section on pluggable providers in the [ManageIQ Developer Setup](http://manageiq.org/docs/guides/developer_setup)
+Redfish provider is installed by default when seting up ManageIQ development
+environment. To start using it, navigate to the `Compute` ->
+`Physical Infrastructure` -> `Providers` and add new provider of type Redfish.
+
+
+## Development environment setup
+
+See the [ManageIQ Developer Setup guide][dev-setup] for help on installing
+prerequistes for ManageIQ development. In this section, we will assume that
+ManageIQ core repo resides in `~/miq/manageiq`, so adjust path accordingly.
+
+   [dev-setup]: https://manageiq.org/docs/guides/developer_setup
+                (ManageIQ developer setup)
+
+Now, to start working Redfish provider, we must first clone the repo:
+
+    $ cd ~/miq
+    $ git clone git@github.com:ManageIQ/manageiq-providers-redfish.git
+    $ cd manageiq-providers-redfish
+
+After this is done, we must perform the initial setup by running
+
+    $ bin/setup
+
+After the initial setup is done, we can test the environment by executing
+
+    $ bundle exec rake
+
+This command should run test suite and return with no error. If this is not
+the case, congratulations to us, since we can start debugging our setup;) If
+everything is green, we are ready start adding ~~bugs~~ features to the
+Redfish provider.
+
+To be able to view and test the changes we are making to the provider, we must
+override the gem sources in `~/miq/manageiq/bundler.d/overrides.rb`. In our
+case, `overrides.rb` should look something like this:
+
+    override_gem "manageiq-providers-redfish",
+      :path => File.expand_path("~/miq/manageiq-providers-redfish")
+
+Do not use relative path in `File.expand_path` call, since this is just
+calling for troubles. To actually install the local gems, run
+
+    $ ( cd ../manageiq && bin/update )
+
+Now, after starting the ManageIQ, the Redfish provider will be gem will be
+installed from local folder instead of being fetched from GitHub.
```

Comment:
Prefer
> Now ManageIQ will use the local manageiq-providers-redfish repository instead of fetching from GitHub.  This will allow you to test changes and debug before your changes have been merged.

Reply:
Will use your suggestion.